components/camel-spring/src/test/resources/org/apache/camel/spring/processor/ThrottlerGroupingTest.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="
+       http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+       http://camel.apache.org/schema/spring http://camel.apache.org/schema/spring/camel-spring.xsd
+    ">
+
+  <bean id="myBean" class="org.apache.camel.spring.processor.SpringThrottlerMethodCallTest"/>
+
+  <camelContext xmlns="http://camel.apache.org/schema/spring">
+    <errorHandler id="dlc" deadLetterUri="mock:dead" type="DeadLetterChannel"/>
+    <route errorHandlerRef="dlc">
+      <from uri="seda:a"/>
+      <throttle timePeriodMillis="1000">
```

Comment:
As found by Willem, the current version of the camel-spring schema requires to put the correlationExpression after the last `to` of this route.

Reply:
this is addressed with xmltype annotation and propOrder attribute of it.